backend/kesaseteli/companies/services.py

```diff
@@ -25,3 +25,20 @@ def get_or_create_company_with_business_id(business_id: str) -> Company:
     company_data = ytj_client.get_company_data_from_ytj_data(ytj_data)
```

Comment:
Not sure if you noticed but there is a YTJClient in shared component, I think you can just use it and get rid of the YTJClient in the KS `companies` app. They are basically the same

Reply:
Yep, I'll create a separate PR for that because I have to set up the shared code in the Dockerfile etc.